src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -238,6 +239,7 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
     const camera = videoOptions[0].options.find(item => item.value === selectedOption) ?? selectedVideoOptions[0];
     setSelectedVideoOptions([camera]);
     switchCameraInput(camera.id);
+    setVideoOptionsOpen(false);
```

Comment:
We set the `optionsOpen` state to `false` here in the `updateOptions` method instead of toggling it off on the click event